scripts/listapranzo.js

```diff
@@ -85,6 +85,29 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var fuzzyMatch = function (dish, menuline) {
+    dish = dish.trim().toLowerCase().replace(" ", "");
+    var key = "";
+    for (var i = 0; i < dish.length; i++)
+      key = key + dish[i] + ".*"
```

Comment:
i file finder fanno una selezione che confermi a vista, proprio perché è molto fuzzy. In questo caso il tipo di interazione è diversa, si dà un comando e non è previsto (nel flusso normale) una ulteriore conferma. 

Inoltre i file finder tipicamente lavorano su alberi di directory dove è normale scrivere qualche lettera di una directory e poi qualche lettera del nome del file (per disambigurare). Qui nessuno scriverà mai "NB" per fare match di "penne all'arrabbiata". Basta tranquillamente scrivere "penne" o "arrabbiata" o anche "arrabb". E' più che sufficiente a scrivere poco, e si fanno meno errori.